src/ObjLoading/Game/T6/BSP/Linker/BSPLinker.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "../BSP.h"
```

Comment:
The project does not use backwards directory traversal for paths 😅 

`Game/T6/BSP/BSP.h` is prefered